code/datums/abilities/critter/writhe.dm

```diff
@@ -1,5 +1,5 @@
 // ----------------------------------
-// spazz out
+// spasm around
```

Comment:
Why not just `writhe around`, as the ability's name is "writhe"?

Reply:
Good point. Done